cluster_agent/settings.py

```diff
@@ -63,6 +69,21 @@ def compute_extra_settings(cls, values):
         if values["SINGLE_USER_SUBMITTER"] is None:
             values["SINGLE_USER_SUBMITTER"] = values["X_SLURM_USER_NAME"]
 
+        assert any(
+            [
+                values["SLURMRESTD_JWT_KEY_PATH"],
+                values["SLURMRESTD_JWT_KEY_STRING"],
+            ]
+        ), "Either SLURMRESTD_JWT_KEY_PATH or SLURMRESTD_JWT_KEY_STRING must be configured"
+
+        if not all(
```

Comment:
What happens if they are both defined?

Reply:
I let it briefly described in the PR description. The idea is to support both cases. In case both are defined then the code will prioritize the use of the key string directly and completely ignore the key path

Reply:
I am uncomfortable with that implicit behavior. I think it should explicitly require one or the other so there is no chance for ambiguity.

Reply:
let me figure it out the changes I will need to make in order to change it

Reply:
so, removing the ambiguity as you say is taking longer than expected. There are a bunch of tests to refactor. Is there any potential problem you see leaving the way it is? I, with honesty, don't see any **if** well documented (which is not the case. I'd need to write the docs for that)

Reply:
according to private conversations with @dusktreader, we agreed that the current implementation is fine once we have clear documentation about the behaviour and the agent logs a warning in case both settings are passed. Both commits [42691ec](https://github.com/omnivector-solutions/cluster-agent/pull/54/commits/42691eccebdcbb3abf5c3c6e207d10d7d4210053) and [74eb9bd](https://github.com/omnivector-solutions/cluster-agent/pull/54/commits/74eb9bd81590b4606b1225f01d0ccea46d617961) handle such requirements.

I'm resolving this conversation. Feel free to review again.